homeassistant/components/vizio/media_player.py

```diff
@@ -380,17 +379,13 @@ def source_list(self) -> list[str]:
     def app_id(self) -> str | None:
         """Return the ID of the current app if it is unknown by pyvizio."""
         if self._current_app_config and self.source == UNKNOWN_APP:
-            return {
-                "APP_ID": self._current_app_config.APP_ID,
-                "NAME_SPACE": self._current_app_config.NAME_SPACE,
-                "MESSAGE": self._current_app_config.MESSAGE,
-            }
+            return self._current_app_config.APP_ID
```

Comment:
This is a functional change: `app_id` should return `str | None`. Does this work for you @raman325?

Reply:
we will likely have to make this dictionary an extra state attribute then. Unfortunately vizio doesn't represent ID's as strings, the name space and message matters too

Reply:
I've reverted the change here and removed the return type annotation from `app_id` to make it pass (for now). As this would be a bigger functional change, it's probably better when it's down separately (later).